src/pages/Auth/Login.jsx

```diff
@@ -1,11 +1,102 @@
-import React from "react";
+import React, { useState } from "react";
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { auth } from "../../firebase-config";
+import { handleChange } from "../../helpers";
+import { useNavigate } from "react-router-dom";
+import Input from "./components/Input";
+import { Brand } from "../../components";
+import { googleSignInHandler } from "./helpers";
+import { FcGoogle } from "react-icons/fc";
 
 const Login = () => {
+  const navigate = useNavigate();
+  const initialCredentialState = {
+    email: "",
+    password: "",
+  };
+  const [credentials, setCredentials] = useState(initialCredentialState);
+  const loginUser = async (e, email, password) => {
+    e.preventDefault();
+    const {
+      user: { providerData, uid },
+    } = await signInWithEmailAndPassword(auth, email, password);
+    console.log(providerData);
+    console.log(uid);
```

Comment:
Remove these logs since it's no longer required

Reply:
Yep